services/PanelService.js

```diff
@@ -18,5 +16,10 @@ module.exports = {
     isValidControlNumber: function(activityYear, agencyCode, respondentId, callback) {
         var query = { 'activity_year': activityYear, 'agency_code': agencyCode, 'respondent_id': respondentId };
         exists('Panel', query, callback);
+    },
+
+    isNotIndependentMortgageCoOrMBS: function(activityYear, agencyCode, respondentId, callback) {
```

Comment:
No, this doesn't care about parent name and requires other lender code to be 1 or 5.